packages/shared/models/board.ts

```diff
@@ -0,0 +1,36 @@
+import mongoose from 'mongoose'
+
+const { ObjectId } = mongoose.Types
+
+const boardSchema = new mongoose.Schema({
```

Comment:
What is our naming convention here? Board or Dashboard? Right now we have dashboard in project

Reply:
Board is better and more descriptive